paywall/src/hooks/useOptimism.js

```diff
@@ -0,0 +1,62 @@
+import { useEffect, useRef, useReducer } from 'react'
+
+import useLocksmith from './useLocksmith'
+import { OPTIMISM_POLLING_INTERVAL } from '../constants'
+
+export default function useOptimism(transaction) {
+  const API = transaction ? `/transaction/${transaction.hash}/odds` : ''
```

Comment:
Let's use lowercase for `api` please. Also, maybe something more descriptive like `apiEndpoint`?